pydantic/env_settings.py

```diff
@@ -13,9 +13,6 @@ class BaseSettings(BaseModel):
     """
     Base class for settings, allowing values to be overridden by environment variables.
 
-    By default environment variables must be upper case and prefixed by APP_ by default. Eg. to override foobar,
-    `export APP_FOOBAR="whatever"`. To change this behaviour set Config options case_insensitive and env_prefix.
-
```

Comment:
@samuelcolvin I wasn't sure if I should include a modified description here -- the defaults seem a little more intuitive now, so maybe this just isn't necessary?

Reply:
makes sense.